napari/layers/_base_layer/model.py

```diff
@@ -92,6 +95,41 @@ def name(self, name):
         self._name = name
         self.events.name()
 
+    @property
+    def indices(self):
+        """Tuple of slice objects for slicing arrays on each dimension."""
+        return self._indices
+
+    @indices.setter
+    def indices(self, indices):
+        if indices == self.indices:
+            return
+        self._indices = indices[-self.ndim:]
+        self._set_view_slice()
+
+    @property
+    def cursor_position(self):
+        """tuple: 2-tuple of cursor in image coordinates. Setter does
+        transformation from canvas coordinates to image coordinates
+        """
+        return self._cursor_position
+
+    @cursor_position.setter
+    def cursor_position(self, cursor_position):
+        if cursor_position is None:
```

Comment:
can you elaborate? right now the value passed will sometimes be none because vispy does that (?!) so I just wanted it to be ignored and not update the `cursor_position` value

Reply:
I will move this check outside the setter